lib/SPIRVProducerPass.cpp

```diff
@@ -112,6 +114,8 @@ enum SPIRVSection {
   kSectionCount
 };
 
+typedef uint32_t SPIRVID;
```

Comment:
Prefer not to introduce this typedef. Just use uint32_t.

Reply:
It does make more explicit what is being returned vs a random number.